File: competitive-events-api/src/main/java/es/jocasolo/competitiveeventsapi/dto/comment/CommentDTO.java
package es.jocasolo.competitiveeventsapi.dto.comment;

import java.io.Serializable;

import es.jocasolo.competitiveeventsapi.dto.DTO;
import es.jocasolo.competitiveeventsapi.dto.user.UserLiteDTO;
import lombok.Getter;
import lombok.Setter;

public class CommentDTO extends DTO implements Serializable {

	private static final long serialVersionUID = 1L;

	@Getter
	@Setter
	private Integer id;

	@Getter
	@Setter
	private String text;

	@Getter
	@Setter
	private String date;
	
	@Getter
	@Setter
	private UserLiteDTO user;

	@Override
	public String toString() {
		return String.format("CommentDTO [id=%s, text=%s]", id, text);
	}

}

File: competitive-events-app/app/src/main/java/es/jocasolo/competitiveeventsapp/dto/comment/CommentDTO.kt
package es.jocasolo.competitiveeventsapp.dto.comment

import com.google.gson.annotations.SerializedName
import es.jocasolo.competitiveeventsapp.dto.HistoryItemDTO
import es.jocasolo.competitiveeventsapp.dto.user.UserDTO
import java.util.*

class CommentDTO(
        @SerializedName("id") var id: String,
        @SerializedName("text") var text: String?,
        @SerializedName("date") var date: Date?,
        @SerializedName("user") var user: UserDTO?
) : HistoryItemDTO() {
}